cellengine/resources/compensation.py

```diff
@@ -30,6 +30,7 @@ class Compensation(DataClassMixin):
             decoder=lambda x: numpy.array(x),
         )
     )
+    _inverted: Any = field(default=None)
```

Comment:
Minor, but this can probably be `Union[None, Dataframe]` or `Optional[Dataframe]`?